README.md

```diff
@@ -813,12 +813,6 @@ classDiagram
     RuleSet --> "*" Comment : aComments
     RuleSet --> "*" Rule : aRules
     URL --> "1" CSSString : oURL
-    ValueList --> "*" CSSFunction : aComponents
-    ValueList --> "*" CSSString : aComponents
-    ValueList --> "*" LineName : aComponents
-    ValueList --> "*" RuleValueList : aComponents
-    ValueList --> "*" Size : aComponents
-    ValueList --> "*" URL : aComponents
```

Comment:
In the class diagram, we'll need to change all those blocks to `… --> "*" Value` accordingly (for all associations that now use the `Value` class). This should also simplify the class diagram a lot.